api/converter/to_pb.go

```diff
@@ -99,156 +112,222 @@ func ToClientsMap(clientsMap map[string][]string) map[string]*api.Clients {
 }
 
 // ToEventType converts the given model format to Protobuf format.
-func ToEventType(eventType types.EventType) api.EventType {
+func ToEventType(eventType types.EventType) (api.EventType, error) {
 	switch eventType {
 	case types.DocumentsChangeEvent:
-		return api.EventType_DOCUMENTS_CHANGED
+		return api.EventType_DOCUMENTS_CHANGED, nil
 	case types.DocumentsWatchedEvent:
-		return api.EventType_DOCUMENTS_WATCHED
+		return api.EventType_DOCUMENTS_WATCHED, nil
 	case types.DocumentsUnwatchedEvent:
-		return api.EventType_DOCUMENTS_UNWATCHED
+		return api.EventType_DOCUMENTS_UNWATCHED, nil
 	default:
-		panic("unsupported event")
+		return 0, fmt.Errorf("%v: %w", eventType, ErrUnsupportedEventType)
```

Comment:
`EventType` is declared as a string. How about changing this line like below?

```suggestion
		return 0, fmt.Errorf("%s: %w", eventType, ErrUnsupportedEventType)
```